### PR TITLE
yamllint 1.11.1 (new formula)

### PR DESCRIPTION
**Pull request by apeschel**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by apeschel** [comment]
Apologies on the original broken build -- I did run `brew audit --strict`, but CI uses `brew audit --online --new-formula yamllint` which catches a few issues the former does not. It was be useful to have the documentation updated accordingly.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/yamllint.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class Yamllint < Formula
+  include Language::Python::Virtualenv
+
+  desc "Linter for YAML files"
+  homepage "https://github.com/adrienverge/yamllint"
+  url "https://github.com/adrienverge/yamllint/archive/v1.11.1.tar.gz"
+  sha256 "56221b7c0a50b1619e491eb157624a5d1b160c1a4f019d64f117268f42fe4ca4"
+
+  depends_on "libyaml"
+  depends_on "python@2"
```
[the remark this answers is not in the document]
Comment (apeschel) [reply Formula/yamllint.rb]: @fxcoudert Didn't notice the python3 compatibility, nice catch.

**Comment by apeschel** [comment]
@fxcoudert Thanks for the review -- I've made some changes corresponding to your requests.

**Comment by fxcoudert** [comment]
Thanks @apeschel for your contribution! Merged with a slightly adjusted test block.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**